[IMP] sales: add event prods to quote templates

ksc-odoo opened the pull request:
PROJECT TASK: https://www.odoo.com/odoo/3835/tasks/3987827?cids=3

removes an admonition tag from a portion of the quote template doc because it is more of a necessary step, than a "tip"

and I added a new TIP admonition to address the recent release note re: the ability to add event-related products to quotation templates in 17. This was a requested addition from VPD.

robodoo commented:
[![Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/9783.png)](https://mergebot.odoo.com/odoo/documentation/pull/9783)

ksc-odoo commented:
Alrighty, @pabr-odoo -- this one is ready for a quick Peer Review from you, whenever you get a chance. It's a very simple add/adjustment to a doc. I moved some content from a TIP admonition into the body of the doc, and added a **new** TIP admonition to address the release note re: the ability to add event-related products to a quotation template (which is now available in Odoo 17). If you have any questions, just let me know. Thanks! 👍

pabr-odoo reviewed (approved):
Hey @ksc-odoo Just tested the new events-related products and it all looks good to me! Passing this on to the next review.

ksc-odoo commented:
Thanks for the quick review, @pabr-odoo -- appreciate it 🙏 -- alrighty, looks like this one is ready for Final Review, @StraubCreative (whenever you get a free moment). Thanks! 👍

StraubCreative reviewed (approved):
LGTM 👍 

@samueljlieber you're up next for tech review.

samueljlieber reviewed (approved):
Thanks for your work @ksc-odoo! Your changes look good to me 👍
.. 
@robodoo r+
